src/data.js

```diff
@@ -0,0 +1,53 @@
+const data = {
```

Comment:
a common practice keep such data as json

Reply:
Thanks for the tip! I'll definitely need this is my current task.

Reply:
Done